.travis.yml

```diff
@@ -58,6 +58,10 @@ before_deploy:
   - source travis/install-conda.sh
   - conda install -y jinja2 # needed to render conda recipe
 
+after_script:
```

Comment:
Keep this change (which won't affect anything if OSX is not in the build matrix)